pages/Documents/RichMessaging/GettingStartedwithRichMessaging/introduction.md

```diff
@@ -560,4 +561,47 @@ Each basic element can have specific style rules defined for it, controlling how
 }
 ```
 
-For a list of Rich Messaging capabilities for each channel, browse or search the table on the [Knowledge Center](https://knowledge.liveperson.com/messaging-channels-messaging-channels-capabilities-comparison.html).
\ No newline at end of file
+### Element Accessibility Attributes
+
+Each basic element can have specific accessibility attributes defined to them. 
+Some of the widely used attributes are listed below. For the entire list of supported attributes, please refer to [this list](https://github.com/LivePersonInc/json-pollock/blob/master/js/schema/accessibilityWeb.json).
+
+#### Properties
+
+| Property Name    | Description                                                                 |
+| :--------------- | :-------------------------------------------------------------------------- |
+| role             | Describes the role of an element for screen readers                         |
```

Comment:
Nit: Consider marking the attribute names up as code.

Reply:
I was just following how the attributes were addressed in other parts in the page. And added the code as an example below.